git/bootstrap.sh

```diff
@@ -1,7 +1,7 @@
 GIT_CONFIG_PATH="$DOTFILE_INSTALL_PATH/gitconfig.symlink";
 
 git_clone() {
-    git clone $1 $2 &>> $LOGDIR/git.log;
+    git clone $1 $2 >> $LOGDIR/git.log;
```

Comment:
I'm not sure why I added the ampersand but I'm guessing it was to log errors.  Is there some value to including `STDERR` in stdout stream and not the log?